src/util.ts

```diff
@@ -0,0 +1,10 @@
+export function seconds_to_display_time(seconds: number | string): string {
+  const d = Number(seconds);
+  const h = Math.floor(d / 3600);
+  const m = Math.floor((d % 3600) / 60);
+  const s = Math.floor((d % 3600) % 60);
```

Comment:
maybe extract `3600` as constant instead of repeating magic nr?

Reply:
Hello, I have enterprized the `seconds_to_display_time` function, please review:
```ts
export function seconds_to_display_time(
  seconds_to_format: number | string,
): string {
  const ONE_HOUR_REPRESENTED_IN_SECONDS: number = 3600;
  const DIVISION_OPERAND_MINUTES = 60;
  const MODULO_OPERAND_SECONDS = 60;

  const seconds_to_format_in_number_representation = Number(seconds_to_format);
  const number_of_hours = Math.floor(
    seconds_to_format_in_number_representation /
      ONE_HOUR_REPRESENTED_IN_SECONDS,
  );
  const number_of_minutes = Math.floor(
    (seconds_to_format_in_number_representation %
      ONE_HOUR_REPRESENTED_IN_SECONDS) /
      DIVISION_OPERAND_MINUTES,
  );
  const number_of_seconds = Math.floor(
    (seconds_to_format_in_number_representation %
      ONE_HOUR_REPRESENTED_IN_SECONDS) %
      MODULO_OPERAND_SECONDS,
  );

  const output_string_values = [];
  if (number_of_hours > 0) {
    output_string_values.push(number_of_hours);
    output_string_values.push('h');
  }

  if (number_of_minutes > 0) {
    output_string_values.push(number_of_minutes);
    output_string_values.push('m');
  }

  if (number_of_seconds > 0) {
    output_string_values.push(number_of_seconds);
    output_string_values.push('s');
  }

  return output_string_values.join('');
}
```

Reply:
Smells like skjatt jippity.